scripts/core/queue.yml

```
@@ -108,6 +115,7 @@ returns:
         - "`propSize != 0 && pPropValue == NULL`"
         - "`pPropValue == NULL && pPropSizeRet == NULL`"
     - $X_RESULT_ERROR_INVALID_QUEUE
```

Missing `:` is required, currently this is treating the bullet on the line below as part of this string rather than an array item.

```suggestion
    - $X_RESULT_ERROR_INVALID_QUEUE:
```

Ends up rendered as this which doesn't make sense:

```cpp
///     - ::UR_RESULT_ERROR_INVALID_QUEUE - "If `hQueue` isn't a valid queue handle or if `propName` isn't supported by `hQueue`."
```